cns/restserver/restserver.go

```diff
@@ -41,6 +45,17 @@ const (
 	detach          = "Detach"
 	// Rest service state identifier for named lock
 	stateJoinedNetworks = "JoinedNetworks"
+	// CNS Snspshot properties
+	cnsNCSnapshotEventStr         = "CNSNCSnapshot"
+	ipConfigurationStr            = "IPConfiguration"
+	localIPConfigurationStr       = "LocalIPConfiguration"
+	primaryInterfaceIdentifierStr = "PrimaryInterfaceIdentifier"
+	multiTenancyInfoStr           = "MultiTenancyInfo"
+	cnetAddressSpaceStr           = "CnetAddressSpace"
+	allowNCToHostCommunicationStr = "AllowNCToHostCommunication"
+	allowHostToNCCommunicationStr = "AllowHostToNCCommunication"
+	networkContainerTypeStr       = "NetworkContainerType"
+	orchestratorContextStr        = "OrchestratorContext"
```

Comment:
can you move this to logger/constants.go ..we already defined for metrics and logs and you can add this for events section